sktime/transformations/compose/_ytox.py

```diff
@@ -177,3 +183,14 @@ def _inverse_transform(self, X, y=None):
             return y.loc[X.index.intersection(y.index)]
         else:
             return y
+
+    @classmethod
+    def get_test_params(cls):
+        """Provide test parameters for the YtoX class."""
+        from sktime.transformations.series.boxcox import BoxCoxTransformer
+        from sktime.transformations.series.exponent import ExponentTransformer
+
+        return [
+            {"subset_index": False, "transformer": ExponentTransformer(power=2)},
```

Comment:
this should also include the setting where no `transformer` is provided, and the empty dict, i.e., the overall default.

Reply:
The changes requested are done